src/codeyCommand.ts

```diff
@@ -45,8 +46,8 @@ export class SapphireMessageResponseWithMetadata {
 
 export type SapphireMessageExecuteType = (
   client: SapphireClient<boolean>,
-  // Message is for normal commands, ChatInputInteraction is for slash commands
-  messageFromUser: Message | SapphireCommand.ChatInputInteraction,
+  // Message is for normal commands, ChatInputCommandInteraction is for slash commands
```

Comment:
Sapphire refactored how commands work and changed the name of the corresponding interaction